app/Console/Commands/AddProductCommand.php

```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace App\Product\Infrastructure\Console;
+namespace App\Console\Commands;
 
-use App\Product\ProductFacade;
+use Src\Product\ProductFacade;
```

Comment:
Src? Why not App?

Reply:
I thought maybe the code was "dirty" if I put the `Product` module inside `/app`, what do you think @Chemaclass ?

![image](https://user-images.githubusercontent.com/6381924/134728533-8d132da0-88c9-4a94-aacd-cb145ee928a3.png)

Reply:
Yeah, it's a good point. I think I kinda like it 🤔 
Let’s leave it like this 👍🏼